eng/scripts/Collect-Changelogs.ps1

```diff
@@ -18,7 +18,10 @@
 param(
```

Comment:
I suspect this script is no longer interesting and can be deleted now with our automation we have in place.